lib/workers/global/autodiscover.ts

```diff
@@ -22,38 +22,33 @@ export async function autodiscoverRepositories(
     return config;
   }
   // Autodiscover list of repositories
-  let discovered = await platform.getRepos();
-  if (!discovered?.length) {
+  const allRepos = await platform.getRepos();
+  if (!allRepos?.length) {
```

Comment:
```suggestion
  let discovered = await platform.getRepos();
  if (!discovered?.length) {
```

rename not required

Reply:
I find this way more readable, but I understand if you want to keep only the necessary changes.